code/modules/mob/living/carbon/human/species_types/lizardpeople.dm

```diff
@@ -21,7 +21,7 @@
 	gib_types = list(/obj/effect/gibspawner/lizard, /obj/effect/gibspawner/lizard/bodypartless)
 	skinned_type = /obj/item/stack/sheet/animalhide/lizard
 	exotic_bloodtype = "L"
-	disliked_food = GRAIN | DAIRY
+	disliked_food = TOXIC | FRIED

```

Comment:
Dude, their `toxic_food` bitfield already contains the `TOXIC` flag.
Look at human/species.dm, open your eyes.

Reply:
This re-wrights over the other disliked food??